client/lib/analytics/survicate.js

```diff
@@ -44,9 +50,20 @@ export function mayWeLoadSurvicateScript() {
 	return config( 'survicate_enabled' );
 }
 
-export function addSurvicate() {
-	const workspaceId = 'e4794374cce15378101b63de24117572';
+/**
+ * Checks if the user is on an anonymous path.
+ * @returns {boolean} True if the user is on an anonymous path, false otherwise
+ */
+export function isUserOnAnonymousPaths() {
+	return [
+		'/log-in',
+		'/setup/onboarding/user',
+		'/log-in/lostpassword',
```

Comment:
What about `/log-in/link` and `/log-in/qr`? Maybe there are more anon paths that I'm missing?

Reply:
Thanks for these additional ones. If we're missing, we'll still catch them in the `calypso_survicate_user_not_available_error` log